src/components/BorrowerProfile/LendCta.vue

```diff
@@ -221,7 +222,7 @@
 							<span
 								class="tw-inline-block tw-align-middle"
 								key="numLendersStat"
-								v-show="statScrollAnimation"
+								v-if="statScrollAnimation"
```

Comment:
👍  Didn't realize we still had `show` instead of `if` here which is required to fully remove the markup from the DOM to isolate the transitions.